libnmstate/nm/ovs.py

```diff
@@ -175,10 +184,17 @@ def _get_bridge_port_info(port_profile, devices_info):
                 port_name, port_setting, port_slave_names
             )
             port_info.update(port_lag_info)
-        if vlan_mode:
-            port_info["vlan-mode"] = vlan_mode
-            port_info["access-tag"] = port_setting.props.tag
 
+        if vlan_mode:
+            nmstate_vlan_mode = NM_OVS_VLAN_MODE_MAP.get(vlan_mode)
+            if not nmstate_vlan_mode:
+                logging.warnings(
+                    f"OVS Port VLAN mode '{vlan_mode}' is not supported yet"
+                )
+            port_info[OB.Port.VLAN_SUBTREE] = {
+                OB.Port.Vlan.MODE: nmstate_vlan_mode,
+                OB.Port.Vlan.TAG: port_setting.props.tag,
```

Comment:
What if there is nothing set? Maybe normalize to `0`? Or just do not add it.

Reply:
The code of NetworkManager `nm_setting_ovs_port_get_tag()` said if not set, return 0.
Changed to use `get_tag()` function.